pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   logging: ^1.1.1
   flutter_web_plugins:
     sdk: flutter
-  uuid: ^3.0.7
+  uuid: ^4.3.3
```

Comment:
A user has just requested this upgrade. Flutter Web is the only plugin to use this package, and I've tested Web on release mode, everything is working fine.